comit/tests/bitcoin_transaction_pattern_e2e.rs

```diff
@@ -1,97 +1,102 @@
-use bitcoincore_rpc::RpcApi;
-use comit::btsieve::bitcoin::{watch_for_created_outpoint, BitcoindConnector};
-use images::coblox_bitcoincore::BitcoinCore;
-use reqwest::Url;
-use std::time::Duration;
-use testcontainers::*;
-use time::OffsetDateTime;
+#[cfg(feature = "test-docker")]
```

Comment:
You could also just cfg the actual test :)

Reply:
I would get clippy warnings...